src/dmd/backend/backconfig.d

```diff
@@ -360,21 +360,6 @@ void out_config_debug(
 
 }
 
-/*************************************
- */
-
-void util_set16()
-{
-    // The default is 16 bits
-    _tysize[TYldouble] = 10;
-    _tysize[TYildouble] = 10;
-    _tysize[TYcldouble] = 20;
-
-    _tyalignsize[TYldouble] = 2;
-    _tyalignsize[TYildouble] = 2;
-    _tyalignsize[TYcldouble] = 2;
-}
```

Comment:
Did you check if dmc still uses this function? If so, it would be worth mentioning in the commit message.

Reply:
There's no way to check this, so I wanted to get @WalterBright 's opinion.

Reply:
DMC is open sourced on the digitalmars github org IIRC, so it can be checked.